gui/qt/seed_dialog.py

```diff
@@ -160,6 +160,13 @@ def on_edit(self):
         self.seed_type_label.setText(label)
         self.parent.next_button.setEnabled(b)
 
+        # to account for bip39 seeds
+        for word in self.get_seed().split(" ")[:-1]:
+            if word not in self.wordlist:
+                self.seed_e.disable_suggestions()
+                break
+            else:
+                self.seed_e.enable_suggestions()
```

Comment:
it might make more sense for the `else` to be one indentation layer "outer"; so that it accompanies the `for` loop instead of the `if`